src/bouquet.c

```diff
@@ -892,42 +892,42 @@ const idclass_t bouquet_class = {
     {
       .type     = PT_BOOL,
       .id       = "maptoch",
-      .name     = N_("Auto-Map to Channels"),
+      .name     = N_("Auto-Map to channels"),
       .off      = offsetof(bouquet_t, bq_maptoch),
       .notify   = bouquet_class_maptoch_notify,
     },
     {
       .type     = PT_BOOL,
       .id       = "mapnolcn",
-      .name     = N_("Map Zero Numbers"),
+      .name     = N_("Map zero numbers"),
```

Comment:
I'd need to see the context of this - I can see it in the bouquet tab, but what does it mean? "Map zero-numbered services"?